src/dependencies.jl

```diff
@@ -864,15 +864,15 @@ macro install(_libmaps...)
                         depsfile_buffer = IOBuffer()
                         println(depsfile_buffer,
                             """
-                            # This is an auto-generated file; do not edit
+                            # This is an auto-generated file; do not edit this file.
```

Comment:
I do not believe this is the case.
I checked out master and did `Pkg.update`  then I checked out this PR and did `Pkg.update` and none of the packages rebuilt.